### DISCUSSION TITLE
How do I pin LazyVim to a specific version?

### DISCUSSION THREAD

**Discussion by JasterV** [LazyVim/LazyVim#5065] [Q&A] [answered]
Hello there :)

I want to pin Lazyvim to a 12.x version since the new updates break my setup that was heavily relying on telescope and other plugins that have been deprecated.

I want to rollback my local version of LazyVim to 12.x and maybe when I have the time adapt my configuration to the new versions.

**Comment by folke** [comment, marked as answer]
Put the below somwhere in your specs.

```lua
{ "LazyVim", version = "13" }
```

Then update with `:Lazy`

**Reply by JasterV** [reply]
thank you it works!

**Reply by mconrado** [reply]
thanks for reply!
